src/app/api/routes/events.py

```diff
@@ -40,6 +40,14 @@ async def fetch_events():
     return await crud.fetch_all(events)
 
 
+@router.get("/past", response_model=List[EventOut], summary="Get the list of all past events")
+async def fetch_past_events():
+    """
+    Retrieves the list of all events and their information
+    """
+    return await crud.fetch_all(events, exclusions={"end_ts": None})
```

Comment:
Isn't it going to be a lot to return? 

At some point we will return hundreds of fire (and other events in the future), donno if the JSon file may take it. 
Shouldn't we add a clause to restrict the scope?

Reply:
Do you mean a maximum number of returned entries?

Reply:
Yes for example, kind of like a pagination.
Or limit the returned results to a department or something. Maybe I am overthinking this.

Reply:
I agree on limiting this to a region, but we don't have the means to do this currently. Or do we? (I might be missing something)

Perhaps limiting the number of entries would be a good idea. I just checked [here](https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.query.Query.limit) it's possible, but if we limit this, we would need to order the results first (to return the most recent). I think it might be better to keep this for another PR

Reply:
ok :+1: